packages/apps/explorer/.eslintignore

```diff
@@ -3,3 +3,5 @@
 !.*.js
 **/dist/**
 **/coverage/**
+**/*.ts
```

Comment:
What's the reason these files are ignored / is there a plan to revisit them later?

Reply:
We have a bunch of React specific linting errors/warnings that I'm not sure how to deal with yet. Especially every function needs to have a return type, which basically for every component is `JSX.Element`. Maybe it'd be better to customize a react set of rules for some projects

Reply:
Got it. As discussed in Slack we'll take it offline and discuss on how to handle these.